webpay/api/base.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import permissions
+
+
+class Permission(permissions.IsAuthenticated):
```

Comment:
Permission seemed a little generic to describe this, I thought it was a generic base class in DRF or something. WebpayPermission or ValidSession maybe?